package.json

```diff
@@ -1,48 +1,34 @@
 {
   "name": "ember-container-query",
   "version": "3.2.0",
+  "private": true,
   "description": "Container queries using Ember modifiers",
-  "keywords": [
-    "container-queries",
-    "container-query",
-    "element-queries",
-    "element-query",
-    "ember-addon",
-    "ember-octane",
-    "emberjs",
-    "glint",
-    "responsive-design"
-  ],
   "repository": {
     "type": "git",
     "url": "https://github.com/ijlee2/ember-container-query.git"
   },
   "license": "MIT",
   "author": "Isaac J. Lee",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
+  "workspaces": [
+    "ember-container-query",
+    "test-app"
+  ],
   "scripts": {
-    "build": "ember build --environment=production",
-    "build:test": "ember build --environment=test",
+    "build": "yarn workspace ember-container-query run build",
+    "build:deploy": "yarn workspaces run build",
```

Comment:
**thought**

I'm not sure why the `build` script (which builds the `addon` package but not the `test-app` package) is needed. I had derived the code from [`@embroider/addon-blueprint`](https://github.com/embroider-build/addon-blueprint).

I wondered, if I could replace the script with `yarn workspaces run build` instead so that I could run the same `build` script for Netlify deployment (documentation).